common/src/main/java/com/sequenceiq/cloudbreak/common/dal/ResourceCrnNameId.java

```diff
@@ -0,0 +1,10 @@
+package com.sequenceiq.cloudbreak.common.dal;
+
+public interface ResourceCrnNameId {
```

Comment:
Basically, I would not like to change this class by anyone, because if we add more fields it can cause more queries. However, the naming convention is ok for me.

Reply:
Thx! If you do not want to change this by anyone, then you should somehow enforce it (with unit test for example), otherwise anyone can change it anytime:)